index.md

```diff
@@ -48,6 +48,8 @@ Documentation is also available online:
 * Tool usage documentation is available [here](tools/latest)
 * Documentation of the various metrics files is available [here](metrics/latest)
 * API documentation for developers is available through [javadoc.io](http://www.javadoc.io/doc/com.fulcrumgenomics/fgbio_2.12).
+* Web utility for validating [Read Structures](https://github.com/fulcrumgenomics/fgbio/wiki/Read-Structures) is available [here](validate-read-structure.html)
```

Comment:
Maybe we should also add a link to the page explaining read structures here?  (And other wiki pages?)

Reply:
The page for the read structure is linked on the same line, but I see what you mean.  I'll defer for now.